action.yml

```
@@ -26,6 +26,9 @@ inputs:
     required: false
     description: Clifus pull request labels
     default: dependencies
+  PUSH_TO_FORK:
+    required: false
+    description: Repository to push the pull request
```

```suggestion
    description: Repository to push the pull request to
```
nitpick, otherwise all good!